nextflow.config

```
@@ -56,6 +56,11 @@ params {
 
     arg_skip_rgi                        = false
     arg_rgi_savetmpfiles                = false
+    arg_rgi_alignmenttool               = 'BLAST'
+    arg_rgi_includeloose                = false
+    arg_rgi_excludenudge                = false
```

If so, then also set `--exclude_nudge` to true in order to keep Loose, Strict and Perfect hits separate in the output.
```suggestion
    arg_rgi_excludenudge                = true
```